### ISSUE TITLE
An "Automatic Progress Value" option for Min/Max progress

### ISSUE THREAD

**Issue by Causese** [WeakAuras/WeakAuras2#1487]
**Is your feature request related to a problem? Please describe.**
It's frustrating that i can't set max/min progress via TSU (dynamically)
1. once i wanted to make a bar appear as full when boss has >0 energy and <50, once he reached 50, i wanted it to make 75 appear as full (e.g set min to 50 and set max to 75) etc.
2. i wanted to set max progress equal to a bosses absorb shield, either with UNIT_AURA (tooltip value) or UNIT_ABSORB_AMOUNT_CHANGED but there's no way to do it

**Describe the solution you'd like**
Basically an "Automatic Progress Value" option in Display, similar to "Automatic Icon".
And then introducing:
minProgress
maxProgress
in TSUs

e.g making it Automatic Progress Value by default and if you untick it, then set min / max shows?

**Describe alternatives you've considered**
overwriting data?

**Additional context**

**Comment by InfusOnWoW** [comment]
I don't see why 2) needs to have any setting of max progress. 

And 1) is easily done with a bit of math.
